vm/src/bench/kernel.rs

```diff
@@ -205,15 +205,16 @@ where
         self.inner_kernel.compute_unsealed_sector_cid(proof_type, pieces)
     }
 
-    // forwarded
+    // NOT forwarded
     fn verify_signature(
         &self,
-        sig_type: SignatureType,
-        signature: &[u8],
-        signer: &Address,
-        plaintext: &[u8],
+        _sig_type: SignatureType,
+        _signature: &[u8],
+        _signer: &Address,
+        _plaintext: &[u8],
     ) -> Result<bool> {
-        self.inner_kernel.verify_signature(sig_type, signature, signer, plaintext)
+        // TODO have this toggleable or give a whitelist of signatures to allow etc. for now, allow everything
+        Ok(true)
```

Comment:
FYI a convention in the actors is to have the signature verification succeed if the signature bytes match the plaintext bytes.